desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserActions/StorageBrowserActions.tsx

```diff
@@ -66,7 +66,7 @@ const StorageBrowserActions = ({
     setLoadingFiles(true);
     rename(selectedFile, newName)
       .then(() => {
-        setRefreshKey((oldKey: number) => oldKey + 1);
+        onSuccessfulAction((oldKey: number) => oldKey + 1);
```

Comment:
So what I meant what that this logic belongs in the parent
Can you move  `(oldKey: number) => oldKey + 1` to the parent component and here just call:
` onSuccessfulAction();`